web/src/buddy_list.ts

```diff
@@ -976,6 +976,26 @@ export class BuddyList extends BuddyListConf {
             is_subscribed_user,
             is_participant_user: this.render_data.all_participant_ids().has(user_id),
         });
+        this.display_or_hide_sections();
+        this.render_section_headers();
```

Comment:
What's the story with adding these here? It seems like this will happen once for each participant when called from `rerender_participants`...

Reply:
Yeah, I guess I will add a `bulk` update function here.